deposit/src/main/java/edu/unc/lib/deposit/normalize/Simple2N3BagJob.java

```diff
@@ -126,6 +128,26 @@ private void populateSimple(Model model, Resource primaryResource, String alabel
 			failJob("Failed to find upload file for simple deposit: " + filename,
 					contentFile.getAbsolutePath());
 		}
+		
+		String checksum = null;
+		String fullPath = contentFile.toString();
+		
+		try {
+			checksum = DigestUtils.md5Hex(new FileInputStream(fullPath));
+			
+			PremisLogger premisDepositLogger = getPremisLogger(getDepositPID());
```

Comment:
This should be an event of the file being uploaded, not the deposit.  You can get that in this case by using primaryResource.toString()